.gitignore

```diff
@@ -24,6 +25,7 @@ acceptance/junit
 acceptance/log
 acceptance/.bundle
 acceptance/.beaker
+acceptance/.byebug_history
```

Comment:
I think the `.byebug_history` on line 15 should cover this one, since it isn't anchored to the root of the tree (doesn't start with `/`). Was this not the case?

Reply:
You're right. I updated that and simplified it some.